### PR TITLE
Rebrand

### PR DESCRIPTION
**Pull request by 0xhedgie**

### CODE REVIEW

File: common/config.ts [hunk 1]
```diff
@@ -1,47 +1,23 @@
+// smartRouterHelper - is it necessary? cannot find usage
+
+// TODO: update name, `cake` -> `sect`?
```
Comment (gedanziger) [inline common/config.ts]: I think it's worth doing, if anything as an exercise to make sure you fully understand the deployment code

File: common/config.ts [hunk 2]
```diff
@@ -1,47 +1,23 @@
...
+  linea: {
+    WNATIVE: '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f',
     nativeCurrencyLabel: 'ETH',
-    v2Factory: '0x1097053Fd2ea711dad45caCcc45EfF7548fCB362',
-    stableFactory: '0x0000000000000000000000000000000000000000',
-    stableInfo: '0x0000000000000000000000000000000000000000',
-    cake: '0x152649eA73beAb28c5b49B26eb48f7EAD6d4c898',
-    smartRouterHelper: '0xdAecee3C08e953Bd5f89A5Cc90ac560413d709E3',
+    cake: '0x92ae51D09765f653B3433Df42d5dba1dde0ab039',
+    smartRouterHelper: '0x4C35fA1249a72190fc2e08fd778d0E15E8bcb586',
```
Comment (gedanziger) [inline common/config.ts]: probably good to TODO updating this

File: projects/router/contracts/libraries/SmartRouterHelper.sol [hunk 3]
```diff
@@ -2,161 +2,16 @@
...
-            amounts[i - 1] = getAmountIn(amounts[i], reserveIn, reserveOut);
-        }
-    }
-
-
-
     /************************************************** V3 **************************************************/
 
-    bytes32 internal constant V3_INIT_CODE_HASH = 0x6ce8eb472fa82df5469c6ab6d485f17c3ad13c8cd7af59b3d4a8026c5ce0f7e2;
+    // f*ck!
```
Comment (gedanziger) [inline projects/router/contracts/libraries/SmartRouterHelper.sol]: It isn't immediately obvious what this is used for / how it's computed. Enhance the comment to explain it's used in the createPool and how the quantity isderived.

File: projects/router/test/SectaSwapSmartRouter.test.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import { assert, expect } from "chai";
+
+
+const WrappedBNB = artifacts.require("exchange-protocol/contracts/libraries/WBNB.sol");
+const SmartRouter = artifacts.require("./SmartRouter.sol");
+
+
+
+contract("SmartRouter", ([admin, bob, carol]) => {
+});
```
Comment (gedanziger) [inline projects/router/test/SectaSwapSmartRouter.test.ts]: The old tests had some stableswap components, but these contracts were changed (if just removing v2/stableswap components) - there should be test coverage

**Review by gedanziger — COMMENTED** [review]
Outside of the stableswap removal and v2 removal, the changes are largely cosmetic. I'll add a bit more detail later today, but I wanted to drop these items at first

**Comment by jac0x** [comment]
Continue on https://github.com/secta-finance/secta-v3-contracts/pull/2

**Pull request closed without merging**